src/renderer/components/Home.vue

```diff
@@ -4,7 +4,7 @@
     <nav class="navbar navbar-default">
       <div class="container-fluid">
         <div class="navbar-header">
-          <a class="navbar-brand">Data Curator</a>
+          <a class="navbar-brand"></a>
```

Comment:
partially addresses OCC-207

Reply:
May add back later now the Data Curator menu is removed for Windows. Will decide in next release 3..5